### DISCUSSION TITLE
Add register endpoints and SDK method

### DISCUSSION THREAD

**Discussion by potros** [directus/directus#5979] [Feature Requests]
it will be very helpful to add register endpoints and SDK method for registration and that will make directus a true competitor in headless CMSs and it will be more easier to build fullstack apps with it with no boilerblate.

**Comment by rijkvanzanten** [comment]
In the meantime, you might be able to rely on the `invite` method / endpoint: https://docs.directus.io/reference/sdk/#invite-a-new-user

**Reply by potros** [reply]
I think that it will be better to open an issue for that what.

what do you think about that @rijkvanzanten ?

**Reply by rijkvanzanten** [reply]
What do you mean?

**Reply by potros** [reply]
Open an issue on the repository so you can add it to future versions of directus like v9.0.0-rc.75 or something like that

**Reply by rijkvanzanten** [reply]
We'll get to that point once enough people upvote the original feature request 😉 

See https://github.com/directus/directus/discussions/3083

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
